### PR TITLE
Massive style cleanup

### PR DESCRIPTION
**Pull request by Arcanemagus**
Adds a basic `setup.cfg` file with configurations for flake8 and pylama, fixes some basic issues with essentially every file including:
* Many, many, whitespace line cleanups
* Several unused variables and imports
* Missing coding and shabang lines
* Minor style fixes to more closely align with PEP8
* Turn `print` into function calls for Python 2/3 compat
* A few minor bugs
  * Things like using an undefined `i` in `stream_limiter_ban_email.py`

### CODE REVIEW

File: killstream/limiterr.py [hunk 1]
```diff
@@ -48,7 +51,7 @@
 from datetime import datetime
 import sys
 import os
-from plexapi.server import PlexServer, CONFIG
+from plexapi.server import PlexServer
```
Comment (Arcanemagus) [inline killstream/limiterr.py]: `CONFIG` isn't being used, I split the `import` of that up.

File: maps/ips_to_maps.py [hunk 2]
```diff
@@ -30,12 +33,12 @@
 import os
 from collections import OrderedDict
 import argparse
-import numpy as np
+# import numpy as np
```
Comment (Arcanemagus) [inline maps/ips_to_maps.py]: This appears to only used in commented out code.

File: notify/twitter_notify.py [hunk 3]
```diff
@@ -14,18 +17,18 @@
 https://gist.github.com/blacktwin/261c416dbed08291e6d12f6987d9bafa
 """
 
-from twitter import *
+from twitter import Twitter, OAuth
```
Comment (Arcanemagus) [inline notify/twitter_notify.py]: It _appears_ these are the only objects being used from here, but I haven't verified this still works.

File: reporting/plex_netflix_check.py [hunk 4]
```diff
@@ -29,15 +32,14 @@
 from xmljson import badgerfish as bf
 from lxml.html import fromstring
 from time import sleep
-import json
```
Comment (Arcanemagus) [inline reporting/plex_netflix_check.py]: This didn't appear to be used, either from the tools or manually looking over the code.

File: utility/find_plex_meta.py [hunk 5]
```diff
@@ -54,12 +57,11 @@ def hash_to_path(hash_str, path, title, media_type, artist=None):
         output = "{} titled: {}\nPath: {}".format(media_type.title(), title, full_path)
     print(output)
 
+
 def get_plex_hash(search, mediatype=None):
     for searched in plex.search(search, mediatype=mediatype):
-        # Remove special characters from name
-        clean_title = re.sub('\W+',' ', searched.title)
```
Comment (Arcanemagus) [inline utility/find_plex_meta.py]: This wasn't being used at all.

File: utility/grab_gdrive_media.py [hunk 6]
```diff
@@ -12,15 +13,11 @@
 from googleapiclient.discovery import build
 from oauth2client.client import OAuth2WebServerFlow
 
-import time, shutil, sys
```
Comment (Arcanemagus) [inline utility/grab_gdrive_media.py]: None of these _appeared_ to be being used.

File: utility/stream_limiter_ban_email.py [hunk 7]
```diff
@@ -295,9 +296,9 @@ def send_notification(to=None, friendly=None, val_cnt=None, val_tot=None, mess=N
         try:
             if act_lst.count(user) >= LIMIT:
                 # Trigger for first and next violation
-                unshare(user) # Remove libraries
-                share(user, BAN) # Share banned library
-                sys.stdout.write("Shared BAN_LIBRARY with user {0}".format(i))
+                unshare(user)  # Remove libraries
+                share(user, BAN)  # Share banned library
+                sys.stdout.write("Shared BAN_LIBRARY with user {0}".format(user))
```
Comment (Arcanemagus) [inline utility/stream_limiter_ban_email.py]: `i` wasn't defined, from the context it appears that `user` was the intended variable.

**Comment by blacktwin** [comment]
Looks good to me. Thanks a lot.

**Pull request merged**